packages/components/src/components/form/bal-input/stories/bal-input.docs.mdx

```diff
@@ -77,6 +77,14 @@ Combine the component with the `bal-field` components to add a label, validation
   <Story id="components-form-input--offer-number-input" />
 </Canvas>
 
+### Simple Phone Number Input
```

Comment:
add the anchor for scrolling before the title like 

`<span id="story--components-form-input--simple-phone-number-input" style={{ opacity: 0 }}></span>`